Author: Rafael Waldo Delgado Doblas <lord.rafa@gmail.com>

This disables the mini_pcie WIFI hardware block pin at boot.

diff -u a/arch/arm/boot/dts/tegra124-jetson-tk1.dts b/arch/arm/boot/dts/tegra124-jetson-tk1.dts
--- a/arch/arm/boot/dts/tegra124-jetson-tk1.dts
+++ b/arch/arm/boot/dts/tegra124-jetson-tk1.dts
@@ -2089,3 +2089,14 @@
 		};
 	};
 };
+
+&gpio {
+	/* Jetson TK1 GPIO_X7_AUD controls PCIE W_DISABLE */
+	pex-w-disable {
+		gpio-hog;
+		gpios = <TEGRA_GPIO(X, 7) GPIO_ACTIVE_LOW>;
+		output-low;
+		line-name = "PEX_W_DISABLE";
+	};
+};
+
